src/app/uzivatel/[id]/ProfileContent.tsx

```diff
@@ -0,0 +1,28 @@
+'use client';
+
+import { FC } from 'react';
+
+import { useAuth } from '../../../context/AuthContext';
+
+type ProfileContentProps = {
+  ownerId: string;
+  ownerName: string | undefined;
```

Comment:
Would you happen to know what this means? It's basically, "I wanted ownerName either a string or EXPLICIT undefined." Maybe you wanted `?` optional.